Hugging face clean history

jig925 opened the pull request:
Added new hugging face boards

jig925 committed:
Hugging Face Boards

Added new hugging face boards

New Branch as old commit history had public keys

jig925 committed:
Update package-lock.json

Mearman commented:
nice!
what do you think to:
- [ ]  push each into its own folder
- [ ] add json for each board
- [ ] add a brief README in each folder for each board (perhaps with a link with the bb web link?)
- [ ] have these been tested with the web editor?
- [ ] (probably as a separate ticket) add these (or versions of) to the main BB repo (as tool worker boards if appropriate)

jig925 committed:
WIP fill mask

jig925 committed:
Moved Boards, added json, refactor

moved boards into their own folders

Added json for each board

WIP

jig925 committed:
Labelling board input handling

jig925 committed:
Fixing Board Inputs

fixed board inputs to not use * for inputs as it does not work on breadboard web

jig925 committed:
Attempting Board Fix

jig925 committed:
Update board.json

jig925 committed:
Fix Board Schema

jig925 committed:
Board Schema Fix

jig925 committed:
Board Fix, Updating README

jig925 committed:
Trying Fix

jig925 committed:
Fix

jig925 committed:
Fixing URL Issues, formatting

jig925 committed:
Update index.ts

jig925 committed:
Updated Model

Updated model used by text generation

jig925 committed:
Fixed Schema

jig925 committed:
Updating Schema

jig925 committed:
Update README.md

jig925 committed:
Updated README, New Boards

Added audio transcript board (WIP)

Added image classification board (WIP)

Tidied up README

Tidied up hugging face boards

jig925 committed:
Schema Bug Board

jig925 committed:
Updating bugged board example

jig925 committed:
Update index.ts

jig925 committed:
Update board.json
